geopandas/tests/test_pandas_methods.py

```diff
@@ -603,6 +603,15 @@ def test_apply_preserves_geom_col_name(df):
     assert result.geometry.name == "geom"
 
 
+def test_df_apply_returning_series(df):
+    # https://github.com/geopandas/geopandas/issues/2283
+    result = df.apply(lambda row: row.geometry, axis=1)
+    assert_geoseries_equal(result, df.geometry, check_crs=False)
+
+    result = df.apply(lambda row: row.value1, axis=1)
+    assert_series_equal(result, df["value1"])
```

Comment:
```suggestion
    assert_series_equal(result, df["value1"].rename(None))
```

attempt to fix the tests